app/src/main/java/com/hamdan/forzenbook/theme/Theme.kt

```diff
@@ -5,104 +5,72 @@ import androidx.compose.foundation.isSystemInDarkTheme
- */
+
 
 @SuppressLint("ConflictingOnColor")
 @Composable
 fun ForzenBookTheme(content: @Composable () -> Unit) {
-    val replacementTypography = ReplacementTypography(
-        defaultFontFamily = FontFamily(
-            Font(R.font.mont_light, FontWeight.Light),
-            Font(R.font.mont_light_ital, FontWeight.Light, FontStyle.Italic),
-            Font(R.font.mont_medium, FontWeight.Medium),
-            Font(R.font.mont_medium_ital, FontWeight.Medium, FontStyle.Italic),
-            Font(R.font.mont_bold, FontWeight.Bold),
-            Font(R.font.mont_bold_ital, FontWeight.Bold, FontStyle.Italic),
-        ),
-        h1 = TextStyle(fontSize = TextSizeValues.large_3),
-        button = TextStyle(fontSize = TextSizeValues.med_3),
-        h3 = TextStyle(fontSize = TextSizeValues.med_1),
-        h4 = TextStyle(fontSize = TextSizeValues.small_3),
-    )
-    val replacementColors: ReplacementColors = if (isSystemInDarkTheme()) {
-        ReplacementColors(
-            darkColors(
-                background = COLOR_LION_YELLOW,
-                primary = BLACK,
-                onPrimary = WHITE,
-                secondaryVariant = WHITE,
-                surface = DARK_GRAY,
-                onSurface = GRAY,
-                onError = Color.Red,
-                onBackground = BLACK,
-            )
+    val replacementTypography = if (LocalConfiguration.current.screenWidthDp <= SMALL_SCREEN_DP) {
+        ReplacementTypography(
+            defaultFontFamily = defaultFontFamily,
+            h1 = TextStyle(fontSize = TextSizeValues.large_2),
+            h2 = TextStyle(fontSize = TextSizeValues.med_2),
+            button = TextStyle(fontSize = TextSizeValues.med_1),
+            h3 = TextStyle(fontSize = TextSizeValues.small_3),
+            h4 = TextStyle(fontSize = TextSizeValues.small_2),
         )
     } else {
-        ReplacementColors(
-            lightColors(
-                background = COLOR_LION_YELLOW,
-                primary = WHITE,
-                onPrimary = Color.Black,
-                secondaryVariant = WHITE,
-                surface = DARK_GRAY,
-                onSurface = GRAY,
-                onError = Color.Red,
-                onBackground = BLACK,
-            )
+        ReplacementTypography(
+            defaultFontFamily = defaultFontFamily,
+            h1 = TextStyle(fontSize = TextSizeValues.large_4),
+            h2 = TextStyle(fontSize = TextSizeValues.large_1),
+            button = TextStyle(fontSize = TextSizeValues.med_3),
+            h3 = TextStyle(fontSize = TextSizeValues.med_1),
+            h4 = TextStyle(fontSize = TextSizeValues.small_3),
         )
     }
+    val replacementColors: ReplacementColors = if (isSystemInDarkTheme()) {
+        ReplacementColors(darkTheme)
+    } else {
+        ReplacementColors(lightTheme)
+    }
+    val adaptiveDimens: Dimens = if (LocalConfiguration.current.screenWidthDp <= SMALL_SCREEN_DP) {
+        //small screens
```

Comment:
Unsure why, but Ktlint didn't run on your commit. The unused import above should have been caught too.

`./gradlew ktLintCheck` will check it for you